Description: This patch fixes some spelling errors found by lintian.
Origin: vendor
Author: Alessandro Ghedini <al3xbio@gmail.com>
Reviewed-by: gregor herrmann <gregoa@debian.org>
Last-Update: 2013-10-23

--- a/lib/XML/LibXML/Document.pod
+++ b/lib/XML/LibXML/Document.pod
@@ -180,7 +180,7 @@
 
   $doc->setEncoding($new_encoding);
 
-This method allows to change the declaration of encoding in the XML declaration
+This method allows one to change the declaration of encoding in the XML declaration
 of the document. The value also affects the encoding in which the document is
 serialized to XML by $doc->toString(). Use setEncoding() to remove the encoding
 declaration.
--- a/lib/XML/LibXML/InputCallback.pod
+++ b/lib/XML/LibXML/InputCallback.pod
@@ -73,7 +73,7 @@
 Callback groups are implemented as a stack (Array), each entry holds a
 reference to an array of the callbacks. For the libxml2 library, the
 XML::LibXML::InputCallback callback implementation appears as one single
-callback group. The Perl implementation however allows to manage different
+callback group. The Perl implementation however allows one to manage different
 callback stacks on a per libxml2-parser basis.
 
 
--- a/lib/XML/LibXML/Parser.pod
+++ b/lib/XML/LibXML/Parser.pod
@@ -230,7 +230,7 @@
 =back
 
 By default XML::LibXML does not process XInclude tags within an XML Document
-(see options section below). XML::LibXML allows to post process a document to
+(see options section below). XML::LibXML allows one to post process a document to
 expand XInclude tags.
 
 =over 4
--- a/lib/XML/LibXML/PI.pod
+++ b/lib/XML/LibXML/PI.pod
@@ -56,7 +56,7 @@
   $pinode->setData( $data_string );
   $pinode->setData( name=>string_value [...] );
 
-This method allows to change the content data of a PI. Additionally to the
+This method allows one to change the content data of a PI. Additionally to the
 interface specified for DOM Level2, the method provides a named parameter
 interface to set the data. This parameter list is converted into a string
 before it is appended to the PI.
--- a/lib/XML/LibXML/RelaxNG.pod
+++ b/lib/XML/LibXML/RelaxNG.pod
@@ -36,11 +36,11 @@
 a validation schema. It is important, that each schema only have a single
 source.
 
-The location parameter allows to parse a schema from the filesystem or a URL.
+The location parameter allows one to parse a schema from the filesystem or a URL.
 
 The string parameter will parse the schema from the given XML string.
 
-The DOM parameter allows to parse the schema from a pre-parsed L<<<<<< XML::LibXML::Document >>>>>>.
+The DOM parameter allows one to parse the schema from a pre-parsed L<<<<<< XML::LibXML::Document >>>>>>.
 
 Note that the constructor will die() if the schema does not meed the
 constraints of the RelaxNG specification.
@@ -50,7 +50,7 @@
 
   eval { $rngschema->validate( $doc ); };
 
-This function allows to validate a (parsed) document against the given RelaxNG
+This function allows one to validate a (parsed) document against the given RelaxNG
 schema. The argument of this function should be an XML::LibXML::Document
 object. If this function succeeds, it will return 0, otherwise it will die()
 and report the errors found. Because of this validate() should be always
--- a/lib/XML/LibXML/Schema.pod
+++ b/lib/XML/LibXML/Schema.pod
@@ -35,7 +35,7 @@
 a validation schema. It is important, that each schema only have a single
 source.
 
-The location parameter allows to parse a schema from the filesystem or a URL.
+The location parameter allows one to parse a schema from the filesystem or a URL.
 
 The string parameter will parse the schema from the given XML string.
 
@@ -47,7 +47,7 @@
 
   eval { $xmlschema->validate( $doc ); };
 
-This function allows to validate a (parsed) document against the given XML
+This function allows one to validate a (parsed) document against the given XML
 Schema. The argument of this function should be a L<<<<<< XML::LibXML::Document >>>>>> object. If this function succeeds, it will return 0, otherwise it will die()
 and report the errors found. Because of this validate() should be always
 evaluated.
--- a/lib/XML/LibXML/Text.pod
+++ b/lib/XML/LibXML/Text.pod
@@ -136,14 +136,14 @@
 
 The more programmer friendly version of replaceData() :)
 
-Instead of giving offsets and length one can specify the exact string (I<<<<<< $oldstring >>>>>>) to be replaced. Additionally the I<<<<<< $all >>>>>> flag allows to replace all occurrences of I<<<<<< $oldstring >>>>>>.
+Instead of giving offsets and length one can specify the exact string (I<<<<<< $oldstring >>>>>>) to be replaced. Additionally the I<<<<<< $all >>>>>> flag allows one to replace all occurrences of I<<<<<< $oldstring >>>>>>.
 
 
 =item replaceDataRegEx( $search_cond, $replace_cond, $reflags )
 
   $text->replaceDataRegEx( $search_cond, $replace_cond, $reflags );
 
-This method replaces the node's data by a C<<<<<< simple >>>>>> regular expression. Optional, this function allows to pass some flags that will
+This method replaces the node's data by a C<<<<<< simple >>>>>> regular expression. Optional, this function allows one to pass some flags that will
 be added as flag to the replace statement.
 
 I<<<<<< NOTE: >>>>>> This is a shortcut for
--- a/LibXML.pod
+++ b/LibXML.pod
@@ -479,7 +479,7 @@
 
   $gdomenode = XML::LibXML->export_GDOME( $node, $deep );
 
-Allows to clone an XML::LibXML node into an XML::GDOME node.
+Allows one to clone an XML::LibXML node into an XML::GDOME node.
 
 
 
